tests/suites/static_analysis/lint_go.sh

```diff
@@ -45,19 +45,19 @@ run_deadcode() {
   if [ -n "${OUT}" ]; then
     echo ""
     echo "$(red 'Found some issues (deadcode):')"
-    echo "\\n${OUT}" >&2
+    echo -e "\\n${OUT}" >&2
     exit 1
   fi
 }
 
 run_misspell() {
   FILES="${2}"
   # shellcheck disable=SC2086
-  OUT=$(misspell -source=go 2>/dev/null ${FILES} || true)
+  OUT=$(misspell -i=gratuitious ${FILES} 2>/dev/null || true)
```

Comment:
For anyone curious about why this exception is needed, see https://github.com/juju/juju/blob/2.8/network/netplan/netplan.go#L191-L193